packages/wallets/integration-tests/menu.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { mockGeneral, mockLoggedIn, setupMocks } from '@deriv/integration';
+import { expect, test } from '@playwright/test';
+import { mockBalance } from './mocks/mockBalance';
+import { mockCryptoConfig } from './mocks/mockCryptoConfig';
+import { mockGetAccountTypes } from './mocks/mockGetAccountTypes';
+import { mockProposalOpenContract } from './mocks/mockProposalOpenContract';
+import mockWalletsAuthorize, { DEFAULT_WALLET_ACCOUNTS } from './mocks/mockWalletsAuthorize';
+
+test.describe('Wallets - Traders Hub', () => {
+    test('render header', async ({ baseURL, page }) => {
+        await setupMocks({
+            baseURL,
+            mocks: [
+                mockGeneral,
+                mockLoggedIn,
+                mockWalletsAuthorize,
+                mockGetAccountTypes,
+                mockCryptoConfig,
+                mockProposalOpenContract,
+                mockBalance,
+            ],
+            page,
+            state: {
+                accounts: DEFAULT_WALLET_ACCOUNTS,
+            },
+        });
+        await page.goto(`${baseURL}/wallets`);
+
+        await expect(page.getByText(`Trader's Hub`)).toBeVisible();
+        await expect(page.getByText(`Cashier`)).toBeVisible();
```

Comment:
```suggestion
        await expect(page.getByText(`Trader's Hub`)).toBeVisible();
        await expect(page.getByText(`Cashier`)).toBeVisible();
```
These checks the top navigation bar which is actually **outside** of wallets 🤔. We already have `VRW1004` account as part of the mock. Why not check if that wallet is actually rendered on screen

Reply:
I get where you're coming from, but I think from the perspective of the wallets squad, they expect when you go to the `/wallets` page that a header is on the page, right?

The nav isn't accessible, but ideally the test should be something like this:

```
    const tradersHubMenuItem = screen.getByRole('link', { name: 'Trader's Hub' });
    expect(tradersHubMenuItem).toHaveAttribute('aria-current', 'page');
```

I think you're right thought that we should also add a test for the VRW1004 account. I'll get that in now.

Reply:
@markwylde-deriv  no.. that is not part of the trader's hub and is bound to change which makes this test pretty flaky

<img width="911" alt="Screenshot 2023-11-21 at 10 38 04 AM" src="https://github.com/binary-com/deriv-app/assets/75345074/855378bf-d5e1-41b9-b4c5-a4e158413c1f">

Reply:
Flakey tests are not tests which fails when UI change (which is expected, thats what they are build for - to detect changes), but tests which are unreliable (e.g. fails 20% of times, while working 80% of times due to e.g. race hazards). So it shouldn't be flakey.

Someone need to fix tests for UI changes, no matter where the tests itselfs live. So personally I do not mind. I expect tests to be failing when UI changes - otherwise, there would be no point in having them at all. They are build to detect changes of UI. They will not fail only when we wont test that bit of UI at all.

Reply:
🤔 I think I disagree @yashim-deriv, because going forward we need to stop thinking of things being part of a big monolithic app.

We need to think of Wallets (and each current package) as being it's own individual app. So, when we think of the wallets page (as a feature) being deployed at wallets.deriv.com with it's own repo, it needs to have the header, main content and footer.

Although, I would agree with you that as it's a shared component, it probably doesn't need an exhaustive test suite in Wallets. Just a few tests to make sure it's there. Then we can fully unit test the layout package.

Anyway, for the sake of getting this merged I've removed the test for now in b1d55cb157bb7048b65f29df9b2fdc1a368866f5.